apisix/cli/ngx_tpl.lua

```diff
@@ -399,7 +399,11 @@ http {
     {% if enable_admin and port_admin then %}
     server {
         {%if https_admin then%}
-        listen {* port_admin *} ssl;
+        {% for _, item in ipairs(ssl.listen) do %}
```

Comment:
I don't like the idea that the admin requires multiple ports.

Reply:
Multiple admin ports are useless. You only need to set one in the API client.

Reply:
I did not change the config `port_admin`. It is still the same as before, only one `port_admin` can be set. 
The idea is to match the specific ip in the config `node_listen`，not for multiple ports.